thunderbird/utils.py

```diff
@@ -51,3 +51,60 @@ def get_url():
             return url
 
     return "https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thunderbird/wps"
+
+
+#######################################
+from wps_tools.output_handling import (
+    rda_to_vector,
+    get_robjects,
+    nc_to_dataset,
+    json_to_dict,
+    txt_to_string,
+)
+import requests
+from bs4 import BeautifulSoup
+
+
+def auto_construct_outputs(outputs):
```

Comment:
Doesn't this method already exist in `wps_tools`? Why is it duplicated here?

Reply:
I will delete it I made it before I put it in wps_tools and forgot to delete